NebulaServer/GameLogic/PlayerManager.cs

```diff
@@ -56,11 +57,15 @@ public Player PlayerConnected(NebulaConnection conn)
             {
                 // Make sure that each player that is currently in the game receive that a new player join so they can create its RemotePlayerCharacter
                 player.SendPacket(new RemotePlayerJoined(newPlayer.Id));
+                // IDEA: Maybe send the entire player data in a packet? 
+                // This way we won't need to send seperate packets for each property of the player
+                player.SendPacket(new PlayerColorChanged(newPlayer.Id, player.PlayerColor));
```

Comment:
Both send packet of color update should be in the mode `ReliableUnordered`

Reply:
Wouldn't that make it possible for the PlayerColorChanged packet to arrive before the RemotePlayerJoined packet?

Reply:
Perhaps it would be better to wait until my changes are merged in, as I fix this issue with a handshake packet.

Reply:
I've merged all the other PRs into master if you want to update work code with it.